modules/drivers/bigquery-cloud-sdk/src/metabase/driver/bigquery_cloud_sdk.clj

```diff
@@ -24,7 +24,7 @@
                                       BigQuery$TableOption BigQueryException BigQueryOptions Dataset DatasetId Field
                                       Field$Mode FieldValue FieldValueList QueryJobConfiguration Schema Table TableId
                                       TableResult]
-           java.util.Collections))
+           (clojure.lang PersistentList)))
```

Comment:
```suggestion
           clojure.lang.PersistentList))
```

Namespace checker is fussy and wants them done this way. Event tho it's the same thing